src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -150,10 +150,24 @@ const getMessageFromBraze = async (
     return canShowPromise;
 };
 
+const FORCE_BRAZE_ALLOWLIST = [
+    'preview.gutools.co.uk',
+    'preview.code.dev-gutools.co.uk',
```

Comment:
Do you want m.code too?

Reply:
Good question! The goal was to not support anywhere public facing which I guess technically m.code is. I think I'll leave it for now, but if it's annoying to do without we can always add down the line.